lib/unobtrusive_flash/controller_mixin.rb

```diff
@@ -37,7 +41,9 @@ def unobtrusive_flash_keys
       [:notice, :alert, :error, :success, :warning]
     end
 
-    class << self
+    module ClassMethods
```

Comment:
I do wonder why these even need to class methods as opposed to making all of these instance methods (testing purposes I suppose?). We could simplify this more by making these instance methods and using the `extend self` pattern.